users/models.py

```diff
@@ -93,14 +93,14 @@ def update_social_profile_from_slack_user(slack_user):
             email=profile.email,
```

Comment:
nit: I think this is more understandable `first_name, last_name = "Tommy Lee Jones".split(maxsplit=1)` that get's you the same result

Reply:
I can't make it that clean unfortunately because there aren't always two values to unpack. I tried something else instead.

Reply:
boooooo ... good call, wasn't thinking of that